[FIX] dev: respect Australian independency (#1010)
odoo/documentation: content/developer/reference/data.rst

  for :term:`external identifiers` (used for creation or update)
* each row thereafter creates a new record

> Here's the first lines of the data file defining Australian and US states

Reviewer: ```suggestion
Here's the first lines of the data file defining country states
```
Keep it generic then ;)

> Here's the first lines of the data file defining Australian and US states

Reviewer 2: ```suggestion
Here's the first lines of the data file defining states
```
while that's not visible on the screenshot (because it's sorted by country code) the file contains way the hell more than just US and AU states (most of which I don't think are really used day-to-day).
